.github/workflows/ci.yml

```diff
@@ -346,7 +349,6 @@ jobs:
           GE_USAGE_STATISTICS_URL: https://qa.stats.greatexpectations.io/great_expectations/v1/usage_statistics
         # Due to the relatively small number of usage_stats tests, we deem it appropriate to test them in their entirely through pytest
         run: pytest --no-sqlalchemy --aws-integration -v tests/integration/usage_statistics
-
```

Comment:
nit: can you restore this blank line. I like having a blank line between jobs and having jobs have no blank lines in them. It makes it easier for me to parse quickly.

Reply:
done